extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -363,11 +379,21 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     // TODO: Check for required and allowed parameters. Probably use
     // validateData, from 3p/3p/js, after noving it someplace common.
     const startTime = Date.now();
-    return getPageLevelParameters_(this.win, this.getAmpDoc(), startTime)
-        .then(pageLevelParameters =>
-        googleAdUrl(this, DOUBLECLICK_BASE_URL, startTime,
-            Object.assign(this.getBlockParameters_(), pageLevelParameters),
-          ['108809080']));
+
+    const pageLevelParametersPromise = getPageLevelParameters_(
+        this.win, this.getAmpDoc(), startTime);
+    const rtcRequestPromise = this.executeRtc_();
+    return Promise.all(
+      [pageLevelParametersPromise, rtcRequestPromise]).then(values => {
+        const pageLevelParameters = values[0];
+        let parameters = Object.assign(
+            this.getBlockParameters_(), pageLevelParameters);
+        if (!!values[1]) {
```

Comment:
you don't actually need this check as assigning null/undefined has no effect.  You can change this to:

const parameters = Object.assign(this.getBlockParameters_(), values[1], pageLevelParameters);